CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ option(PROFILING_ENABLED "Whether to enable google-perftools linker flags" OFF)
 option(CCACHE_ENABLED "Whether to enable compiler caching, if available" ON)
 option(CGAL_ENABLED "Whether to enable the CGAL library" ON)
 option(LSD_ENABLED "Whether to enable the LSD library" ON)
+option(CURL_ENABLED "Whether to enable curl for auto-download of resources" ON)
```

Comment:
Should we call this option by its feature, e.g. `AUTODOWNLOAD_ENABLED`, instead of by the dependency, in case we later switch to a different downloading util than curl?

Reply:
Yes, this is better.